### ISSUE TITLE
[Feature request] Add ability to export diagram image to clipboard/file

### ISSUE THREAD

**Issue by grug** [mattpocock/sextant#32]
Big Sextant fan!

It would be fantastic if we had the ability to export an image of our Sextant diagrams straight to clipboard/file so that we could do things like drop them into Github issues or share with peers (sort of like how Excalidraw's "copy as PNG to clipboard" does it).

**Comment by mattpocock** [comment]
Yes, I'd love this too. I've also had a request from a PM internally to export to PDF, though this might be trickier.

My current focus is to work on plugins until we get a decent plugin story for a few major use cases of Sextant - SDK's, express endpoints, testing etc.

I'd love a PR on this from a friendly neighbourhood OSS-er who has done this kind of thing before. Marking this as Help Wanted.

**Comment by mattpocock** [comment]
A thought - moving to making the scenario diagrams use SVG would make this issue trivial. I think that's worth thinking about - we may want to move to a draggable, Excalidraw SVG-style view instead of the current linear one at some point.
